app/src/main/java/camp/nextstep/edu/calculator/MainActivity.kt

```diff
@@ -1,28 +1,74 @@
 package camp.nextstep.edu.calculator
 
 import android.os.Bundle
+import android.widget.Toast
 import androidx.appcompat.app.AppCompatActivity
 import camp.nextstep.edu.calculator.databinding.ActivityMainBinding
+import camp.nextstep.edu.domain.calculator.*
 
 class MainActivity : AppCompatActivity() {
+
     private lateinit var binding: ActivityMainBinding
 
+    private val calculator = Calculator()
+
+    private val expression = Expression { displayExpression ->
+        binding.textView.text = displayExpression
+    }
+
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         binding = ActivityMainBinding.inflate(layoutInflater)
         setContentView(binding.root)
 
-        //next selection 뭐시기...
-
-        binding.button0.setOnClickListener { binding.textView.text = "0" }
-        binding.button1.setOnClickListener { binding.textView.text = "1" }
-        binding.button2.setOnClickListener { binding.textView.text = "2" }
-        binding.button3.setOnClickListener { binding.textView.text = "3" }
-        binding.button4.setOnClickListener { binding.textView.text = "4" }
-        binding.button5.setOnClickListener { binding.textView.text = "5" }
-        binding.button6.setOnClickListener { binding.textView.text = "6" }
-        binding.button7.setOnClickListener { binding.textView.text = "7" }
-        binding.button8.setOnClickListener { binding.textView.text = "8" }
-        binding.button9.setOnClickListener { binding.textView.text = "9" }
+        binding.button0.setOnClickListener { appendExpression(Num(0)) }
+        binding.button1.setOnClickListener { appendExpression(Num(1)) }
+        binding.button2.setOnClickListener { appendExpression(Num(2)) }
+        binding.button3.setOnClickListener { appendExpression(Num(3)) }
+        binding.button4.setOnClickListener { appendExpression(Num(4)) }
+        binding.button5.setOnClickListener { appendExpression(Num(5)) }
+        binding.button6.setOnClickListener { appendExpression(Num(6)) }
+        binding.button7.setOnClickListener { appendExpression(Num(7)) }
+        binding.button8.setOnClickListener { appendExpression(Num(8)) }
+        binding.button9.setOnClickListener { appendExpression(Num(9)) }
+
+        binding.buttonPlus.setOnClickListener { appendExpression(Operators.of("+")) }
```

Comment:
```suggestion
        binding.buttonPlus.setOnClickListener { appendExpression(Operators.Plus }
```

of를 이용하는 경우 조금 더 연산을 진행하게 되네요
이 경우에는 꼭 of가 필요한 경우가 아니고, 바로 선언해주는 것이 더욱 좋을 것 같습니다!